Remotion/Data/DomainObjects/Persistence/Rdbms/SqlServer/Model/Building/SqlStorageTypeInformationProvider.cs

```diff
@@ -25,6 +25,8 @@
 using Remotion.ExtensibleEnums;
 using Remotion.Utilities;
 
+using DateOnlyConverter = Remotion.Utilities.DateOnlyConverter;
```

Comment:
where is the conflict coming from that we need an alias?

Reply:
System.ComponentModel.DateOnlyConverter